s3repo/view.py

```diff
@@ -0,0 +1,90 @@
+"""View for working with the repositories on S3."""
+
+import os
+
+from flask import jsonify
+from flask import render_template
+from flask import Response
+from flask import request
+from flask.views import View
+
+
+class S3View(View):
+    """View for working with S3 according to the REST model."""
+
+    def __init__(self, model):
+        self.model = model
+
+    @staticmethod
+    def response_message(message, status):
+        """Generate response with a message."""
+        response = jsonify({'message': message})
+        response.status_code = status
+        return response
+
+    @staticmethod
+    def __readable_size(size):
+        readable_size = ''
+        for unit in ['', 'Ki', 'Mi', 'Gi']:
+            if abs(size) < 1024.0:
+                readable_size = "%3.1f %sB" % (size, unit)
+                return readable_size
+            size /= 1024.0
```

Comment:
Add some `return` in case of all iterations didn't go inside `if`.

Reply:
Done.